microservices-connector/cmd/router/main.go

```diff
@@ -146,21 +163,32 @@ func executeStep(step *gmcv1alpha3.Step, graph gmcv1alpha3.GMConnector, input []
 	return callService(step, serviceURL, input, headers)
 }
 
-func handleSwitchNode(route *gmcv1alpha3.Step, graph gmcv1alpha3.GMConnector, input []byte, headers http.Header) ([]byte, int, error) {
+func handleSwitchNode(
+	route *gmcv1alpha3.Step,
+	graph gmcv1alpha3.GMConnector,
+	input []byte,
+	headers http.Header,
+) ([]byte, int, error) {
 	var statusCode int
 	var responseBytes []byte
 	var err error
-	stepType := "serviceUrl"
+	stepType := ServiceURL
 	if route.NodeName != "" {
-		stepType = "node"
+		stepType = ServiceNode
 	}
 	log.Info("Starting execution of step", "type", stepType, "stepName", route.StepName)
 	if responseBytes, statusCode, err = executeStep(route, graph, input, headers); err != nil {
 		return nil, 500, err
 	}
 
 	if route.Dependency == gmcv1alpha3.Hard && !isSuccessFul(statusCode) {
```

Comment:
another instance of gmc -> mc for future

Reply:
@mkbhanda sure, got it